src/components/ui/Facets/FacetTypes/NestedLinkItem.tsx

```diff
@@ -52,7 +52,7 @@ function NestedLinkItem(item: NestedLinkItemProps) {
 		<li className="hawk-facet-rail__facet-list-item hawkFacet-group">
 			<div className="hawkFacet-group__inline">
 				<button
-					onClick={() => item.onValueSelected(hierarchyValue, false)}
+					onClick={() => hierarchyChildren.length <= 0 ? item.onValueSelected(hierarchyValue, false) : null}
```

Comment:
If the button is not clickable in this case why do we need to show it?

Reply:
This button will be available to expand list of child but not for self search.